docs/concepts/access-control-concept.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 1
+---
+
+# Decentralized Access Control
+
+## Overview
+
+Lit Protocol provides developers with a decentralized access control layer that can be used to [encrypt](/v3/sdk/access-control/encryption#technical-walkthrough) content for private and permissioned storage on the open Web. Lit provides utilities (via the Lit SDK) for encrypting and decrypting content client-side, while [access control conditions (ACCs)](/v3/sdk/access-control/evm/basic-examples) are used to define who (or under what conditions) the data can be decrypted.
```

Comment:
the links should be relative, and should not include the version!

For example: `../sdk/access-control/encryption#technical-walkthrough`

Reply:
best practice from docusaurus here: https://docusaurus.io/docs/versioning#link-docs-by-file-paths